elyra/pipeline/handlers.py

```diff
@@ -92,4 +94,28 @@ async def post(self, *args, **kwargs):
         self.set_status(200)
         self.set_header("Content-Type", 'application/json')
         self.finish(json_msg)
-        self.flush()
+
+
+class PipelineConfigHandler(HttpErrorMixin, APIHandler):
+    """Handler to expose method calls to retrieve pipelines editor configuration"""
+
+    valid_resources = ["palette", "properties"]
+
+    @web.authenticated
+    def get(self, resource):
+        print(f'>>> retrieving {resource}')
```

Comment:
Please remove or convert to an appropriate log statement.

Reply:
My bad, left over of troubleshooting cis